amenity-points.mss

```diff
@@ -3084,4 +3092,4 @@
       trunk/marker-height: 6;
     }
   }
-}
+}
```

Comment:
It looks like you have accidentally removed new line at the end of file, this is technical detail, but please bring it back.